ui/packages/components/src/utils/date.ts

```diff
@@ -1,4 +1,28 @@
 import { format } from 'date-fns';
+import dayjs from 'dayjs';
+import calendar from 'dayjs/plugin/calendar';
+import durationPlugin from 'dayjs/plugin/duration';
+import localizedFormat from 'dayjs/plugin/localizedFormat';
+import { default as relativeTimePlugin } from 'dayjs/plugin/relativeTime';
```

Comment:
Could we use date-fns instead, since we want to remove dayjs? Would this do the exact same thing?

import { formatDistanceToNow } from 'date-fns';
export function relativeTime(d) {
  return formatDistanceToNow(new Date(d), { addSuffix: true });
}